### ISSUE TITLE
Check Deliverables [Didem]

### ISSUE THREAD

**Issue by zudiay** [bounswe/2021SpringGroup5#218]
I want to make sure that the deliverables work without any errors

**Comment by zudiay** [comment]
I pulled the master code to my local and thanks to Docker I run the code with only one command.  I checked the Dockerfile and docker-compose.yml file. I think it's very clean. It's working without problem. I first tried to run the code without Docker and it failed although I installed the requirements (due to some alias modules, i think). And then I run Docker and it worked perfectly. Lesson learned: Isolation is crucial!

**Comment by zudiay** [comment]
We had some issues on the server. Some apps were not workings, daily quote app I wrote was one of them. They were working when I pulled from master and run with docker, and the server was doing exactly the same thing. There were no error logs, we could not figure out why. Then we closed the server and opened again and the problem was gone. Later, I talked to some people experienced in docker and learned the caches sometimes cause problems like these. It would be nice to use --no-cache option.
